tests/v2/unit/adapters/torch/lightning/modules/datasets/__init__.py

```diff
@@ -0,0 +1,3 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
```

Comment:
some license headers have this extra line `#`, while some don't. it would be good to have consistency

Reply:
I fixed this for the files in that PR.